src/include/duckdb/common/tree_renderer/yaml_tree_renderer.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "duckdb/main/profiling_node.hpp"
+#include "duckdb/common/tree_renderer.hpp"
+#include "duckdb/common/render_tree.hpp"
+
+namespace duckdb {
+class LogicalOperator;
+class PhysicalOperator;
+class Pipeline;
+struct PipelineRenderNode;
+
+class YAMLTreeRenderer : public TreeRenderer {
+public:
+	explicit YAMLTreeRenderer() {
+	}
+	~YAMLTreeRenderer() override {
+	}
+
+public:
+	string ToString(const LogicalOperator &op);
+	string ToString(const PhysicalOperator &op);
+	string ToString(const ProfilingNode &op);
+	string ToString(const Pipeline &op);
+
+	void Render(const LogicalOperator &op, std::ostream &ss);
+	void Render(const PhysicalOperator &op, std::ostream &ss);
+	void Render(const ProfilingNode &op, std::ostream &ss) override;
+	void Render(const Pipeline &op, std::ostream &ss);
+
+	void ToStreamInternal(RenderTree &root, std::ostream &ss) override;
+	bool UsesRawKeyNames() override {
+		return true;
```

Comment:
Is this required for the YAML rendering? This means nodes like `__estimated_cardinality__` are rendered as such, instead of being rendered as the more readable `"Estimated Cardinality"`